### ISSUE TITLE
Support of parametrized fields in ResponseProjection

### ISSUE THREAD

**Issue by lmartellotto** [kobylynskyi/graphql-java-codegen#153]
Hi,

First, thanks for this project! My team and I want to use your project for testing our GraphQL API with RestAssured. Your generated GraphQL client-side does exactly what we are looking for :) Moreover, the automation with Maven plugin is perfect! 

However, I have a question about ResponseProjection. 

Here, my simple schema:
```
type Query {
    itemById(id: Int!): Item!
}

type Item {
  name: String
  tags (max:Int): [String]
}
```

Generated **OperationRequest** and **ResponseProjection** allow me to write request as follow:

```java
ItemByIdQueryRequest query = new ItemByIdQueryRequest.Builder()
                                                          .setId(1)
                                                          .build();
        
ItemResponseProjection fields = new ItemResponseProjection()
                                                          .name()
                                                          .tags();
GraphQLRequest request = new GraphQLRequest(query, fields);
```

... which generates GraphQL following query:
```
{"query":"query { itemById(id: 1){ name tags } }"}
```

But, my question is: how to give "max" parameter to the "tags" fields? And so, how to generate this query for example:
```
{"query":"query { itemById(id: 1){ name tags(max: 10) } }"}
```

It seems that we cannot generate this query, but maybe I forget something? 

Thank you for your time,
Laura

**Comment by kobylynskyi** [comment]
HI @lmartellotto 
I am happy to hear that you found this library helpful.
Unfortunately, as of today, parametrized fields are not supported in ResponseProjection classes. 
But I can definitely work on this sometime in the future.

**Comment by lmartellotto** [comment]
Thanks for your reply.

It's great to know it could be available in the future :) Maybe I can help with a PR... Are you open to that ? I will investigate and keep you in touch if I start anything.

**Comment by kobylynskyi** [comment]
@lmartellotto I will start working on that if you don't mind.

**Comment by kobylynskyi** [comment]
Released in [1.8.0](https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v1.8.0)

**Comment by lmartellotto** [comment]
@kobylynskyi I forget to say thank you!! I currently use it in my integration tests and it works well! So, thank you for your work and the quick release 🚀 👍
